immuni_common/models/mongoengine/temporary_exposure_key.py

```diff
@@ -33,6 +33,7 @@ class TemporaryExposureKey(EmbeddedDocument, Serializable):
     transmission_risk_level: Enum = EnumField(TransmissionRiskLevel)
     rolling_start_number: int = IntField(required=True)
     rolling_period: int = IntField(required=False)
+    country_of_interest: str = ListField(required=False)
```

Comment:
If it's a list, shouldn't it be `countries` instead of `country`?
```suggestion
    countries_of_interest: str = ListField(required=False)
```